src/Model/ChannelTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Model;
+
+use Doctrine\ORM\Mapping as ORM;
+
+trait ChannelTrait
```

Comment:
This means we have to ask the user of the plugin to change his channel mapping also. I am not sure we need this, do we?

Reply:
About the relation, 1 channel may have 1 configuration, and 1 configuration may be used by multiple channels. So I believe the owner has to be the channel, hasn't it ?

Reply:
Oh, but with the Locale interfering in relation, indeed the relation will change a bit

Reply:
> About the relation, 1 channel may have 1 configuration, and 1 configuration may be used by multiple channels. So I believe the owner has to be the channel, hasn't it ?

Yes, that is correct. Do you think we should use a join table instead?

Reply:
I believe we require a full entity to make the join. It will contain `Channel`, `LocaleCode` and `Configuration`